### ISSUE TITLE
Importing MessageSerializer without Faust is Broken

### ISSUE THREAD

**Issue by SRserves85** [marcosschroh/python-schema-registry-client#47]
**Describe the bug**
When Importing the non-Faust `MessageSerializer`, I get Errors Related to the Faust Serializer

**To Reproduce**
1. I create a new virtualenv
2. `pip install python-schema-registry-client`
3. `from schema_registry.serializers.message_serializer import MessageSerializer`
4. Should Error Out with `AttributeError: 'NoneType' object has no attribute 'Codec'`

NOTE: Importing it this way also produces the same error:
`from schema_registry.serializers import MessageSerializer`

**Expected behavior**
I should be able to import the MessageSerializer without touching `faust_serializer.py`

**Comment by SRserves85** [comment]
Due to the restricted access to post PR's here is the fix I was going to apply to `schema_registry/serializers/__init__.py`:

```python
try:
    import faust  # noqa
    del faust  # noqa
    from schema_registry.serializers.faust_serializer import FaustSerializer  # noqa
except ImportError:  # pragma: no cover
    pass
from schema_registry.serializers.message_serializer import MessageSerializer  # noqa
```

**Comment by marcosschroh** [comment]
Hi @SRserves85 ,

Thanks for reporting it. Question: what do you mean with restricted access to post PR's? Can't you send a `PR`?

**Comment by SRserves85** [comment]
Can you please add me as a contributor? I don't have access to post pr's for this repo, but am more then happy to contribute.

Also if you have anyone asking about using this for spark/making this pickleable, I wrote a custom version with minimal changes that is threadsafe (has to start new connections per thread). Happy to post my tiny rewrite of `SchemaRegistryClient` as well as an example since I'm sure quite a few people are trying to use it with `pyspark`.


In the mean time, I definitely don't have access to post pr's as I keep getting this:
`ERROR: Permission to marcosschroh/python-schema-registry-client.git denied to SRserves85.`

**Comment by marcosschroh** [comment]
I am happy to receive your PRs. Seems that you are trying to push to the repository directely and that is not allowed.

 What you have to do:

1. Fork the repository
2. Create a branch and push (to your respository)
3. Create a pull request using the GitHub UI.

If we merge your PR you are automatically added as a contributor.

Let me know if you have any trubbles

**Comment by SRserves85** [comment]
I did an have a PR up

**Comment by marcosschroh** [comment]
closed by https://github.com/marcosschroh/python-schema-registry-client/pull/49

**Comment by m0baxter** [comment]
I think the fix broke it for people using faust. I am getting this error:

**Comment by marcosschroh** [comment]
@SRserves85 Thanks for reporting it. I fixed it in version `1.2.10`
